addressbook/mainnet.pb

```diff
@@ -620,4 +620,285 @@
 
```

Comment:
This concerns me a bit having it now. And this is a release PR. Maybe move the addressbook update for some later task or if someone requires it. Wouldn't risk braking any user here even if the chance is really slight.